Integrations/integration-Malwr.yml

```diff
@@ -0,0 +1,346 @@
+                soup = None
+                is_malicious = None
+                md5 = None
+            else:
+                status = 'complete'
+                s = self.session
+                req = s.get(self.url + RESULT_URL % (analysis_id, ), headers=self.headers)
+                soup = BeautifulSoup(req.content, 'html.parser')
+                is_malicious = 'malicious' in str(soup)
+                soup_str = str(soup)
+                start_index = soup_str.find(MD5_PREFIX_STR)
+                if start_index == -1:
+                    md5 = None
+                else:
+                    start_index += len(MD5_PREFIX_STR)
+                    md5 = soup_str[start_index : start_index + 32]
+
+            return status, is_malicious, soup, md5
+
+
+    if 'username' in demisto.params() and 'password' in demisto.params():
+        username = demisto.params()['username']
+        password = demisto.params()['password']
+    else:
+        username = None
+        password = None
+
+    malwr = MalwrAPI(
+        url=demisto.params()['server'],
+        username=username,
+        password=password
+    )
+
+    entry = {
+        'Type': entryTypes['note'],
+        'ContentsFormat' : formats['text'],
+        'ReadableContentsFormat' : formats['text']
+    }
+
+    if __name__ == '__main__':
+        if demisto.command() == 'test-module':
+            demisto.results('ok')
+            sys.exit(0)
+
+        elif demisto.command() == 'malwr-submit':
+            file_id = demisto.args()['fileId']
+            filepath = get_file_path(file_id)
+            res, soup = malwr.submit_sample(filepath)
+            if 'analysis_link' in res:
+                analysis_id = res['analysis_link'].split('/')[-2]
+
+                message = 'File submitted: %s%s\n' % (MAIN_URL, res['analysis_link'])
+                message += 'MD5: %s\n' % (res['md5'], )
+                message += 'Analysis ID: %s' % (analysis_id, )
+
+                entry['Contents'] = str(soup)
+                entry['HumanReadable'] = message
+                entry['EntryContext'] = {'Malwr.Submissions(val.Id==obj.Id)': {'Id': analysis_id, 'Md5': res['md5'], 'Status': 'pending'}}
+
+            else:
+                entry['HumanReadable'] = res
+
+        elif demisto.command() == 'malwr-status':
+            analysis_id = demisto.args()['analysisId']
+            status, data, soup = malwr.get_status(analysis_id)
+            if status == 'complete':
+                message = 'The analysis is complete, you can view it at: %s%s.' % (MAIN_URL, data)
+            elif status == 'pending':
+                message = 'The analysis is still in progress.'
+            else:
+                message = 'Error: the specified analysis does not exist.'
+
+            entry['Contents'] = str(soup)
+            entry['HumanReadable'] = message
+            entry['EntryContext'] = {'Malwr.Submissions(val.Id==obj.Id)': {'Id': analysis_id, 'Status': status}}
```

Comment:
Can't we update the file itself with malicious prop instead of adding a new entry to context?